src/hfx_libint_wrapper.F

```diff
@@ -12,384 +12,410 @@
+      MARK_USED(work_forces)
+      MARK_USED(a_mysize)
+      CPABORT("This CP2K executable has not been linked against the required library libint.")
+#endif
 
-!****************************************************************************!
-!****************************************************************************!
-!***                                                                      ***!
-!***  WHAT FOLLOWS IS CODE TO PROVIDE STUB ROUTINES IN ABSENCE OF __LIBINT **!
-!***                                                                      ***!
-!****************************************************************************!
-!****************************************************************************!
+   END SUBROUTINE cp_libint_get_derivs
 
-CONTAINS
+   FUNCTION get_ssss_f_val(lib)
+      TYPE(cp_libint_t) :: lib
+      REAL(KIND=dp) :: get_ssss_f_val
 
-! **************************************************************************************************
-!> \brief ...
-!> \param lib ...
-!> \param max_am ...
-! **************************************************************************************************
-   SUBROUTINE initialize_libint(lib, max_am)
-      TYPE(lib_int)                                      :: lib
-      INTEGER                                            :: max_am
+#if(__LIBINT)
+      get_ssss_f_val = lib%prv(1)%f_aB_s___0__s___1___TwoPRep_s___0__s___1___Ab__up_0(1)
+#else
+      MARK_USED(lib)
+      MARK_USED(get_ssss_f_val)
+      CPABORT("This CP2K executable has not been linked against the required library libint.")
+#endif
 
-      CHARACTER(LEN=*), PARAMETER :: routineN = 'initialize_libint', &
-         routineP = moduleN//':'//routineN
+   END FUNCTION
 
+   SUBROUTINE cp_libint_init_eri(lib, max_am)
+      TYPE(cp_libint_t)               :: lib
+      INTEGER                         :: max_am
+#if(__LIBINT)
+      CALL libint2_init_eri(lib%prv, max_am, C_NULL_PTR)
+#else
       MARK_USED(lib)
       MARK_USED(max_am)
-      CPABORT("This CP2K executable has not been linked against the required library libint (HFX).")
-
-   END SUBROUTINE initialize_libint
-
-! **************************************************************************************************
-!> \brief ...
-!> \param deriv ...
-!> \param max_am ...
-! **************************************************************************************************
-   SUBROUTINE initialize_libderiv(deriv, max_am)
-      TYPE(lib_deriv)                                    :: deriv
-      INTEGER                                            :: max_am
+      CPABORT("This CP2K executable has not been linked against the required library libint.")
+#endif
+   END SUBROUTINE
 
-      CHARACTER(LEN=*), PARAMETER :: routineN = 'initialize_libderiv', &
-         routineP = moduleN//':'//routineN
+   SUBROUTINE cp_libint_set_contrdepth(lib, contrdepth)
+      TYPE(cp_libint_t) :: lib
+      INTEGER           :: contrdepth
+      lib%prv(1)%contrdepth = contrdepth
```

Comment:
I believe an `#if(__LIBINT)` is missing here.

Reply:
is fixed now!